workflow/defaults/places.yaml

```diff
@@ -0,0 +1,31 @@
+# This file sets default values for data and executable locations.
+# See the schema/places.yaml for details.
+default_places: &default_places
+  EXPROOT: !error Please select a EXPROOT directory.
+  HOMEgfs:     !calc ( tools.realpath(tools.abspath("../../")) )
+  HOMEcrow:    !calc tools.crow_install_dir()
+  HOMEobsproc_prep: !expand "{doc.platform.BASE_GIT}/obsproc/obsproc_global.v3.2.3"
```

Comment:
This should be "obsproc_prep_RB-5.2.0" instead of "obsproc_global.v3.2.3" for HOMEobsproc_prep.